spring-cloud-consul-binder/src/main/java/org/springframework/cloud/consul/binder/ConsulSendingHandler.java

```diff
@@ -50,9 +50,8 @@ protected void handleMessageInternal(Message<?> message) {
 		}
 
 		Object payload = message.getPayload();
-		if (payload instanceof byte[])
-		{
-			payload = Arrays.toString((byte[])payload);
+		if (payload instanceof byte[]) {
```

Comment:
*UNRELATED* Formatter changed this and I left the adjustment in as it was out of line w/ code style previously